encrypted-media/scripts/playback-persistent-license-events.js

```diff
@@ -47,20 +54,28 @@ function runTest(config,qualifier) {
                 _events.push(event.messageType + '-response');
                 return _mediaKeySession.update(response);
             }).then(test.step_func(function() {
-                _events.push('update-resolved');
+                _events.push(event.messageType + '-response-resolved');
                 if (event.messageType === 'license-release') {
-                    checkEventSequence( _events,
-                                    ['generaterequest',
-                                        ['license-request', 'license-request-response', 'update-resolved'], // potentially repeating
-                                        'keystatuseschange',
-                                        'playing',
-                                        'remove-resolved',
-                                        'keystatuseschange',
-                                        'license-release',
-                                        'license-release-response',
-                                        'closed-attribute-resolved',
-                                        'update-resolved' ]);
-                    test.done();
+                    setTimeout(test.step_func(function() {
```

Comment:
Yeah, I think if you make that `test.step_timeout(function() {` and adjust the parens on the other side you should be fine.